Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -36,6 +36,9 @@ open class MapView: UIView {
     /// Controls the addition/removal of annotations to the map.
     public internal(set) var annotations: AnnotationOrchestrator!
 
+    // TODO: the naming might be slightly confusing, as the annotations object already covers most cases. Need to check if they should be merged or not. Currently their implementation is pretty different.
```

Comment:
I would love to see the interface be merged. Implementations could still be separated by using a wrapper.

Reply:
So the access would be something like `annotations.viewAnnotations` ? 
One other reason I left it separated, is because on Android the rest of the annotations are managed through a plugin, so there the view annotations will be accessed separately from them. Although I don't mind placing them under the orchestrator in our case.

Reply:
If we do it at all, it'd be nice to achieve some kind of consistency with the existing annotations APIs. So maybe something like `annotations.viewAnnotationManager` to mimic the existing factory methods for the other annotation managers.

That said, it's more important to be consistent with Android, so we would only do this on iOS if we also do it there.